### ISSUE TITLE
ARC 23.10.3a: Issue with network interfaces

### ISSUE THREAD

**Issue by TineUser** [AuxXxilium/arc#600]
After updating ARC from 23.08.2 to 23.10.3a and restoring the config from 23.08.2 I see the following error when booting:

![image](https://github.com/AuxXxilium/arc/assets/77736099/c6e4af2e-1ccb-4921-ac96-9b607065bbc3)

**Comment by MartinKuhl** [comment]
I am not able to connect to DSM web interface after the 23.10.3 update. In the bootloader section everything is working as normal but than in DSM mode no network activity is visable.

**Comment by AuxXxilium** [comment]
https://github.com/AuxXxilium/arc/issues/599

**Comment by TineUser** [comment]
For what can I create config Backups or is ARC doing these backups when I'm updating ARC? I don't want to configure ARC again and again when updating.

And what can I do now If I already restored the config?

And will this be fixed?

**Comment by AuxXxilium** [comment]
if you restored a config, flash clean loader

**Comment by TineUser** [comment]
It looks like DSM isn't booting. Will the ARC configurator boot?

And will this be fixed?

**Comment by TineUser** [comment]
Why do you close this issue although it hasn't been fixed?

**Comment by AuxXxilium** [comment]
there is no issue. 

do not restore config and clean flash your stick or change your bootimage.

**Comment by TineUser** [comment]
I don't think that there's no issue! Why does this bug happen when restoring a config backup? And why won't this be fixed so that ARC automatically corrects the configuration regarding this bug?

**Comment by AuxXxilium** [comment]
because of changes in config! i you join discord you will read everything in releasenotes.
